### PR TITLE
Upgrade SDK version in CI and docker

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Keep version in-sync with .travis.yml
```
Comment (jakobr-google) [inline Dockerfile]: Nit: Should we add 'and app/.travis.yml' ?

Comment (isoos) [reply Dockerfile]: Not sure if `app/.travis.yml` is used for anything. I wanted to check it and remove, but never got around it.

Comment (jakobr-google) [reply Dockerfile]: :)

Comment (kevmoo) [reply Dockerfile]: FYI - https://github.com/dart-lang/mono_repo

But that tool has been updated to use a new kind of file.

I'll fix...

**Comment by isoos** [comment]
> Or .32? #1032 

I don't feel comfortable switching the dartdoc version right before our release, without any time for testing it. The current version has been checked a few times, I'd give a bit more time for the next version.

**Comment by kevmoo** [comment]
ack

**Pull request merged**